README.md

```diff
@@ -1,11 +1,97 @@
-# service-ca
-Coming soon...
+# OpenShift Service CA Operator
 
-Controller to mint and manage serving certificates for Kubernetes services.
+This operator runs the following OpenShift service-serving certificate controllers:
```

Comment:
Can we get rid of `service-serving` terminology? I find just "runs the following OpenShift controllers:" easier to understand and read.